AsyncDisplayKit/ASCellNode.mm

```diff
@@ -120,10 +121,10 @@ - (void)setLayerBacked:(BOOL)layerBacked
   ASDisplayNodeAssert(!layerBacked, @"ASCellNode does not support layer-backing.");
 }
 
-- (void)setNeedsLayout
+- (void)__setNeedsLayout
```

Comment:
Is this intended to be overriding the ASDisplayNode base class method?  Seems fine, just curious how this operates at runtime / why it is essential.